command/connect/envoy/bootstrap_tpl.go

```diff
@@ -262,7 +262,9 @@ const bootstrapTemplate = `{
     {{- end }}
   },
   {{- if .StatsSinksJSON }}
-  "stats_sinks": {{ .StatsSinksJSON }},
+  "stats_sinks": [
```

Comment:
https://github.com/hashicorp/consul/pull/16511/files#r1123725950

Move the brackets into the template file